packages/frontend/src/hooks/user/useJoinOrganizationMutation.ts

```diff
@@ -1,11 +1,7 @@
-import {
-    ApiError,
-    CreateOrganization,
-    UpdateOrganization,
-} from '@lightdash/common';
+import { ApiError } from '@lightdash/common';
 import { useMutation, useQueryClient } from 'react-query';
+
```

Comment:
how is this new line added? through linting?

Reply:
must have been me